### PR TITLE
[semver:patch] SDI-50: ✨ Add in java postgres executor

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

**Review by dazoakley — DISMISSED** [review]
LGTM

**Comment by swilson-mojhub** [comment]
What's the value in added int the `java Postgres executor` ? 

The pipeline is generic? Does this impact Typescript or Node services?

File: src/commands/wait_till_ready_postgres.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+parameters:
+  port:
+    type: integer
+    default: 5432
+  host:
+    type: string
+    default: "localhost"
+steps:
+  - run:
+      name: Wait for port to be ready
```
Comment (dazoakley) [inline src/commands/wait_till_ready_postgres.yml:12]: I'd maybe rename this step `Wait for postgres to be ready`?

Comment (petergphillips) [reply src/commands/wait_till_ready_postgres.yml:12]: agreed, it was a copy and paste from the wait till ready executor :grin:

**Comment by petergphillips** [comment]
> What's the value in added int the `java Postgres executor` ?
> 
> The pipeline is generic? Does this impact Typescript or Node services?
As you can see from https://github.com/ministryofjustice/hmpps-circleci-orb/tree/main/src/executors we try to share common code so that people don't need to re-invent the wheel.  There is already a java localstack postgres executor, this one just adds one without localstack.  It also adds a helpful wait command since curl isn't appropriate for a postgres endpoint

**Review by dazoakley — APPROVED** [review]

**Pull request merged**